docs/source/reference/config.rst

```diff
@@ -1,116 +1,287 @@
 .. _config-yaml:
 
 Advanced Configurations
-===========================
+=======================
 
 You can pass **optional configurations** to SkyPilot in the ``~/.sky/config.yaml`` file.
 
 Such configurations apply to all new clusters and do not affect existing clusters.
 
 .. tip::
 
-  Some config fields can be overridden on a per-task basis through the :code:`experimental.config_overrides` field. See :ref:`here <task-yaml-experimental>` for more details.
+  Some config fields can be overridden on a per-task basis through the ``experimental.config_overrides`` field. See :ref:`here <task-yaml-experimental>` for more details.
 
-Spec: ``~/.sky/config.yaml``
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Syntax
+------
 
```

Comment:
"Below is the configuration syntax and some example values. See detailed explanations under each field."